tutorials/snapcraft/snap-a-python-application.md

```diff
@@ -54,57 +54,83 @@ Survey
  - Intermediate
  - Proficient
 
-## Getting familiar with the project
+## Get familiar with the project
 Duration: 2:00
 
 This project is a **Python** application. It has a very simple project layout with an installation and test script.
 
 ### Introspect the code
 
-Simply run:
+To be able to create the snap, we should also be a little familiar with the code, how it gets built and how it is run.
+
+First, we should fetch the source from its repository:
 
 ```bash
-$ git clone https://github.com/reorx/httpstat
-$ cd httpstat
-$ ls
-[…]
-httpstat.py
-setup.py
+git clone https://github.com/reorx/httpstat
 ```
 
-The two interesting files in that directory are:
-  - `httpstat.py` corresponding to the main (and only!) source file, containing the code. You can execute it directly once it’s made executable.
-  - `setup.py`, which is a standard python installation script file. Running setup will build and install the project on the target host. The good news is that snapcraft has full support for projects following this best python practice.
+If we take a look inside the directory
 
-You will note that this project has no `requirements.txt`, meaning that there is no non standard library module used and that needs to be pulled in at build time. If that were the case, snapcraft would have handle this for you, pulling the correct dependencies via pip, itself.
+```bash
+tree httpstat
+```
 
-Do not hesitate to open the files to get familiar with them. Once this is done, you can now remove that `httpstat` directory.
+...we can see that it is a flat file structure with only a few files:
 
+```
+httpstat/
+├── httpstat.py
+├── httpstat_test.sh
+├── LICENSE
+├── Makefile
+├── README.md
+├── screenshot.png
+└── setup.py
 
+```
+
+For now, the two interesting files in that directory are:
+  - `httpstat.py` corresponding to the main (and only!) source file, containing the code. You can execute it directly once it’s made executable.
+  - `setup.py`, which is a standard Python installation script file. Running setup will build and install the project on the target host. The good news is that snapcraft has full support for projects following this best practice for Python.
+
+You will note that this project has no `requirements.txt`, meaning that there is no non-standard library module used and that needs to be pulled in at build time. If that were the case, snapcraft would have handle this for you, pulling the correct dependencies via pip, itself.
```

Comment:
i will add 'file' after requirements.text which I thinks resolves it, but I will tidy up the rest of the words too